eva-server/src/main/java/uk/ac/ebi/eva/server/ws/GeneWSServer.java

```diff
@@ -60,7 +63,9 @@ public QueryResponse getVariantsByGene(@PathVariable("geneIds") List<String> gen
                                            @RequestParam(name = "polyphen", required = false) String polyphenScore,
                                            @RequestParam(name = "sift", required = false) String siftScore,
                                            @RequestParam(name = "exclude", required = false) List<String> exclude,
-                                           HttpServletResponse response) {
+                                           @RequestParam(name = "annotation-vep-version", required = false) String annotationVepVersion,
+                                           @RequestParam(name = "annotation-vep-cache-version", required = false) String annotationVepCacheVersion,
```

Comment:
There is another annotation related parameter called `annot-ct`, so please use `annot-`as prefix to make them more homogeneous.

Applies to other methods in this class and `RegionWSServer`.